app/uploaders/logo_uploader.rb

```diff
@@ -19,12 +19,11 @@ def default_url(*args)
     "/fallback/default_logo.jpg"
   end
 
-  # Process files as they are uploaded:
-  # process scale: [200, 300]
-  #
-  # def scale(width, height)
-  #   # do something
-  # end
+  process scale: [200, 200]
```

Comment:
this naming should be specific to a version, such a small or medium.  `scale` doesn't tell anything